src/services/service.ts

```diff
@@ -99,8 +99,19 @@ export function getProjectFilteredApps(
     return get(`app/min?teamIds=${projectIds.join(',')}${chartOnlyQueryParam}`);
 }
 
-export function getAvailableCharts(queryString?: string, options?: APIOptions): Promise<{ code: number, result: Chart[] }> {
+export function getAvailableCharts(queryString?: string, page_offset?: number, page_size?: number, options?: APIOptions ): Promise<{ code: number, result: Chart[] }> {
     let url = `${Routes.CHART_AVAILABLE}/discover/`;
+
+
+    if (page_offset>=0 && page_size){
+        if(queryString){
+            queryString = queryString + "&offset=" + page_offset + "&size=" + page_size
```

Comment:
Line no 107-113 can be written as "queryString = `${queryString? 'queryString':'?'}&offset=${page_offset}&size=${page_size}`"